spec/fabricators/web_hook_fabricator.rb

```diff
@@ -8,87 +8,168 @@
   verify_certificate true
   active true
 
-  transient post_hook: WebHookEventType.find_by(name: "post")
+  transient post_created_hook: WebHookEventType.find_by(name: "post_created"),
```

Comment:
I think a pattern like this would be better since then we don't have to do 5 DB calls every time we fabricate:

```ruby
  after_build do |web_hook, transients|
    event_types =
      WebHookEventType.where(name: %w[post_created post_edited post_destroyed post_recovered])
    web_hook.web_hook_event_types = [
      transients[:post_created_hook] || event_types.find { |e| e.name == "post_created" },
      transients[:post_edited_hook] || event_types.find { |e| e.name == "post_edited" },
      transients[:post_destroyed_hook] || event_types.find { |e| e.name == "post_destroyed" },
      transients[:post_recovered_hook] || event_types.find { |e| e.name == "post_recovered" },
    ]
  end
```